### ISSUE TITLE
SVG icons display properly on the macOS version of Strongbox, but on iOS, the icon appears blank. 

### ISSUE THREAD

**Issue by ghqe** [strongbox-password-safe/Strongbox#683]
SVG icons display properly on the macOS version of Strongbox, but on iOS, the icon appears blank.

**Comment by ghqe** [comment]
https://1password.com/img/redesign/logo-light-bg.svg

**Comment by strongbox-mark** [comment]
Hi @ghqe - This is a bug of sorts... Basically macOS natively supports loading and displaying SVG images but iOS doesn't. I hadn't come across this issue before because I never used SVGs for icons but I can see that it does look good on macOS but blank on iOS...

I think the solution here is probably to take a PNG snapshot of the SVG on macOS when saving a new SVG icon which would then also work on iOS.

Alternatively we'd need to import a third party library to have SVG support on iOS though perhaps Apple will add this support natively soon. 

In the meantime, unfortunately, the work around is to save your SVG to a nicely sized PNG/JPG and set your icons using this.

**Comment by emeryj6** [comment]
Also just realized this issue today, been chasing down svgs to replace some pngs and make them sharper and smaller in filesize. Then I found out on iOS they’re just blank 🙁

**Comment by strongbox-mark** [comment]
Yes, aware of this issue, will take your comment as a +1 - Unforunately Apple hasn't yet made a convenient way to do this, but there might be a workaround to investigate.

**Comment by ghqe** [comment]
Please check Raivo OTP.
This app support svg very well.
https://github.com/raivo-otp/ios-application
A native, lightweight and secure time-based (TOTP) & counter-based (HOTP) password client built for iOS.

**Comment by ghqe** [comment]
What is going on?
